rsmtool/analyzer.py

```diff
@@ -883,7 +920,7 @@ def filter_metrics(self,
                                                                      'kappa',
```

Comment:
This is where we'll move wtkappa to the previous list to report unrounded values.